tests/r/FormatRBearTest.py

```diff
@@ -129,3 +140,9 @@
     valid_files=(good_file_6,),
     invalid_files=(bad_file_3,),
     settings={'r_max_expression_length': '25'})
+
+FormatRBearDefaultSettingsTest = verify_local_bear(
```

Comment:
How about adding 1 more test that check good_file_1 and good_file_7 at once, just to check that if we dont provide `r_keep_blank_lines` option, those 2 file are both valid?